src/js/components/image-loading.jsx

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 
-export function ImageLoading() {
+export function ImageLoading({accentColor} = {}) {
+    const bounceStyle = {};
+
+    if (accentColor) {
+        bounceStyle.backgroundColor = `#${accentColor}`;
+    }
+
     return <div className='image-overlay'>
                <div className='three-bounce spinner'>
-                   <div className='bounce1'></div>
-                   <div className='bounce2'></div>
-                   <div className='bounce3'></div>
+                   <div className='bounce1'
```

Comment:
We could refactor this part to remove the duplication, but I'm not sure it's worth the trouble?